### DISCUSSION TITLE
Display FAQs links in category tree

### DISCUSSION THREAD

**Discussion by Adipsys** [thorsten/phpMyFAQ#2657] [Q&A]
Hello,

I'm quite struggling findind an easy way to do this.
I have this category tree and I'd like to display a link to the faqs for each sub categories (HM : APIs, Exploitation etc etc)

![image](https://github.com/thorsten/phpMyFAQ/assets/29979292/fa47b4fa-d85e-4e57-b7ba-beaa03274940)

What would be the better way to do it ?  Modifying buildCategoryList in the CategoryHelper or maybe the show.html template ?
I'm on v3.2.1

edit: actually the idea would be to have the same as the view action of the back but in the front. An accordion showing every faq for each categories.

Thanks :)

**Comment by thorsten** [comment]
I don't really understand your issue. :-)

There is already a link to the FAQs or do I mis-understand you?

**Comment by Adipsys** [comment]
Sorry, I probably explained badly.
Actually the tree you see above is only parents categories ("documentations techniques") and childrens categories ("HM : ...").
Behind the childrens I have many faqs for each.

As I have many faqs I'd like to have a tree in the front with quite the same behaviour of the one we have in the back meaning clicking on a parent category would show the faq in it.

I think I can handle the JS part of the accordion but I'm struggling to get the tree data with : 
* Parent 1
  * Children 1
    * Faq 1
    * Faq 2
  * Children 2
    * Faq 1
    * Faq 2
etc etc
* Parent 2
  * Children 1
etc etc

Hope that's clearer :)
I didn't open a feature request as it might be specific to my usage and not sure if this could be an interesting pmf feature (to configure the front display of the faqs).

**Comment by thorsten** [comment]
ah, now I understand it. That'a not that easy to implement. It's quite a complex database query over different tables including checking permissions. Might be quite slow.

**Comment by Adipsys** [comment]
hum ok this is why I was struggling with it then :)
I'll continue searching to see how I could achieve this.
